### ISSUE TITLE
how to apply this code into binary classfication（如何应用此代码到时间序列二分类）

### ISSUE THREAD

**Issue by wei-jin-fang** [KimMeen/Time-LLM#136]
How to apply this code to time series binary classification, I applied this code to a dataset with time step 40 and dimension 10, but I found that the original code did not provide a classification part, I tried to modify the output layer by spreading the final output again after softmax activation function but I found that the classification was correctly poor, asking if there is any other way to make the modification
（如何应用此代码到时间序列二分类，我将此代码应用到了时间步长40，维度10的数据集上，但是我发现原始代码并未提供分类部分，我尝试修改输出层，将最后输出再次展平，经过softmax激活函数，但是我发现分类正确效果较差，询问是否有其他方法进行修改）

**Comment by WangYipu2002** [comment]
您是用来预测趋势吗，请问之后效果有提示吗

**Comment by wei-jin-fang** [comment]
你好是用在分类的，但是我需要替换大模型输出后的一些层之后是有成效的，
但是我发现似乎随着大模型的体量变大很容易过拟合，导致分类偏向一侧极端，不知道在论文中测试过程有无一些缓解方法，例如dropout？



在 2025-01-22 14:47:27，"wyp-ucas" ***@***.***> 写道：

您是用来预测趋势吗，请问之后效果有提示吗

**Comment by kwuking** [comment]
> 你好是用在分类的，但是我需要替换大模型输出后的一些层之后是有成效的，
> 但是我发现似乎随着大模型的体量变大很容易过拟合，导致分类偏向一侧极端，不知道在论文中测试过程有无一些缓解方法，例如dropout？
> 
> 
> 
> 在 2025-01-22 14:47:27，"wyp-ucas" ***@***.***> 写道：
> 
> 您是用来预测趋势吗，请问之后效果有提示吗
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>

你好，感谢对我们工作的关注，如果采用在分类问题上出现过拟合的现象，可能需要对LLM进行lora finetune，使得LLM能够更好的适应时序分类数据。
